src/test/java/uk/nhs/prm/repo/ehrtransferservice/database/TransferServiceTest.java

```diff
@@ -0,0 +1,226 @@
+package uk.nhs.prm.repo.ehrtransferservice.database;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.prm.repo.ehrtransferservice.builders.ConversationRecordBuilder;
+import uk.nhs.prm.repo.ehrtransferservice.database.enumeration.ConversationTransferStatus;
+import uk.nhs.prm.repo.ehrtransferservice.database.model.ConversationRecord;
+import uk.nhs.prm.repo.ehrtransferservice.exceptions.ConversationIneligibleForRetryException;
+import uk.nhs.prm.repo.ehrtransferservice.repo_incoming.RepoIncomingEvent;
+import uk.nhs.prm.repo.ehrtransferservice.services.ConversationActivityService;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.nhs.prm.repo.ehrtransferservice.database.enumeration.ConversationTransferStatus.*;
+
+@ExtendWith(MockitoExtension.class)
+class TransferServiceTest {
+    @Mock
+    private TransferRepository transferRepository;
+
+    @Mock
+    private ConversationActivityService activityService;
+
+    @InjectMocks
+    private TransferService transferService;
+
+    private final ConversationRecordBuilder conversationRecordBuilder = new ConversationRecordBuilder();
+
+    private static final String NHS_NUMBER = "9798547485";
+    private static final String SOURCE_GP = "B45744";
+    private static final String NEMS_MESSAGE_ID = "2d74a113-1076-4c63-91bc-e50d232b6a79";
+    private static final String DESTINATION_GP = "A74854";
+    private static final String NEMS_EVENT_LAST_UPDATED = "2023-10-09T15:38:03.291499328Z";

```

Comment:
Do we care about this any more?

Reply:
From what I can tell, yes. These are the fields that we use to populate the builder in the `createRepoIncomingEvent` method that is used for each of these tests.

Reply:
Will be removed as part of PRMT-4860.